server/StudySharp.API/Requests/TheoryBlocks/RemoveTheoryBlockByIdRequest.cs

```diff
@@ -0,0 +1,7 @@
+namespace StudySharp.API.Requests.TheoryBlocks
+{
+    public class RemoveTheoryBlockByIdRequest
+    {
+        public int Id { get; set; }
```

Comment:
![image](https://user-images.githubusercontent.com/45018119/134785015-9c30a7ba-8c87-4d4b-96c6-cb92b114e7fa.png)

Reply:
Do the same but to RemoveTheoryBlockByIdRequest accordingly